frontend/src/components/components.css

```diff
@@ -17,3 +17,17 @@
     border-radius: 15px;
     border: 2px solid rgb(0, 95, 204);
 }
+
+.navbar-tabs {
+    padding: 1rem 1rem 0 0.5rem !important;
```

Comment:
I prefer using `em` and `rem` over `px` because it is responsive.
The usage of `!important` here is a shortcut. I am not very familiar with bootstrap's classname-method of css formatting, so I took a shortcut. I will fix this so that this class doesn't even need to exist (I will use bootstrap builtins)